src/resonanceReconstruction/rmatrix/legacy/resolved/SpinGroup/SingleLevelBreitWigner/src/evaluate.hpp

```diff
@@ -46,7 +46,7 @@ void evaluate( const Energy& energy,
   // calculate the resulting cross sections
   result[ this->elasticID() ] += factor * components.elastic;
   result[ this->captureID() ] += factor * components.capture;
-  if ( components.hasFission() ) {
+  if ( this->hasFission() ) {
```

Comment:
I like that `hasFission` moves here, I think that makes more sense.

Reply:
Yeah, it works better in my opinion.